### PR TITLE
Add Readme file

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Add Readme file

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+1. Clone the repository:
+```bash
+git clone https://github.com/your-username/your-repo-name.git
+cd your-repo-name
+```
+
+2. Install required Python libraries:
+```bash
+pip install PyGithub python-dotenv
```
Comment (CsatariGergely) [inline Readme.md]: Add a `requirements.txt` and use `pip install -r requirements.txt`

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- JSON file containing committer affiliations
+
+## Setup
+
+1. Clone the repository:
+```bash
+git clone https://github.com/your-username/your-repo-name.git
+cd your-repo-name
+```
+
```
Comment (CsatariGergely) [inline Readme.md:26]: Add optional steps to create a virtual env.

**Review by CsatariGergely — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (Aryan-Singla) [reply Readme.md]: I have updated the readme and added the requirements.txt

File: Readme.md (hunk 2 above)
Comment (Aryan-Singla) [reply Readme.md:26]: python -m venv venv
source venv/bin/activate

is this you want sir

Comment (Aryan-Singla) [reply Readme.md:26]: To activate the virtual environment

Comment (CsatariGergely) [reply Readme.md:26]: Yes, thanks.

Comment (Aryan-Singla) [reply Readme.md:26]: Sir I will add them in few minutes

**Pull request merged**